src/test/kotlin/com/terraformation/backend/accelerator/db/ModuleEventStoreTest.kt

```diff
@@ -333,4 +363,32 @@ class ModuleEventStoreTest : DatabaseTest(), RunsAsUser {
       eventPublisher.assertEventPublished(ModuleEventScheduledEvent(workshop, newRevision))
     }
   }
+
+  @Nested
+  inner class Delete {
+    @Test
+    fun `throws exception if event does not exist`() {
+      assertThrows<EventNotFoundException> { store.delete(EventId(-1)) }
+    }
+
+    @Test
+    fun `throws exception if no permission to manage events`() {
+      every { user.canManageModuleEvents() } returns false
+
+      val eventId = insertEvent()
+      assertThrows<AccessDeniedException> { store.delete(eventId) }
+    }
+
+    @Test
+    fun `deletes the event`() {
+      val event1 = insertEvent()
+      val event2 = insertEvent()
+
+      assertTrue(eventsDao.findAll().isNotEmpty())
```

Comment:
The assertion failure message would be more useful if this was something like
```kotlin
assertNotEquals(emptyList<ModuleEventsRow>(), eventsDao.findAll(), "Events before deletion")
```
And similarly for the other two assertions below. Testing for equality with a value will mean that if the assertion fails, the test output includes the expected and actual values, and including a message will make it easier to see which assertion failed if you're looking at the output from a CI build.